lib/rspec/rails/example/view_example_group.rb

```diff
@@ -149,6 +149,22 @@ def _include_controller_helpers
 
         helper(*_default_helpers)
 
+        app = ::Rails.application
+        if app.respond_to?(:routes)
```

Comment:
Is there a version of rails that doesn't respond to routes? We only do defensive programming like this if theres a version inconsistency, and even then it's often better to make it a specific to the version of Rails, and often better to conditionally define a method e.g.

```Ruby
included do
  ...
  setup_view_helpers
  ...
end

if ::Rails::VERSION < n
  def setup_view_helpers
    ...
  end
else
  def setup_view_helpers
    ...
  end
end
```

And this pretty much goes for all the if statements in this PR